app/Actions/Service/StartService.php

```
@@ -42,19 +42,37 @@ public function handle(Service $service, bool $pullLatestImages = false, bool $s
+        }
+
+        $compose = data_get($service, 'docker_compose', []);
+        $network = data_get($service, 'destination.network');
+
+        if (blank($network)) {
+            return [];
+        }
+
+        $commands = [];
+        $safeNetwork = escapeshellarg($network);
+        $serviceNames = data_get(Yaml::parse($compose), 'services', []);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Symfony\Component\Yaml\Yaml' in method 'serviceNetworkConnectCommands'. (undefined)

(StaticAccess)